types/test.ts

```diff
@@ -96,7 +96,7 @@ agent
 agent.close((err: any) => { throw err; });
 
 function test1() {
-    const req = chai.request(app).get('/');
+    const req: ChaiHttp.Request = chai.request(app).get('/');
```

Comment:
I think this type addition is a good example! But couldn't we just do the following instead?

   ```suggestion
    const req: request.SuperAgentRequest = chai.request(app).get('/');
```
after importing superagent, of course. There is currently quite a lot of confusion around the difference between chai-http and superagent and I fear that aliasing one of their types in our namespace will add to that confusion, where people will assume that it is different when it is not.

Reply:
Well, the biggest problem is that then we have to make `superagent` a peer dependency though?
I kinda see this as reducing confusion, because we are explicitly defining that a `ChaiHttp.Request` is equivalent to a `SuperAgentRequest`, and anyone who takes a look at the source code will know that at a glance